packages/core/integration-tests/test/react-refresh.js

```diff
@@ -0,0 +1,144 @@
+import assert from 'assert';
+import path from 'path';
+import {
+  bundler,
+  inputFS as fs,
+  getNextBuild,
+  defaultConfig,
+  sleep
+} from '@parcel/test-utils';
+import getPort from 'get-port';
+import JSDom from 'jsdom';
+import nullthrows from 'nullthrows';
+
+const inputDir = path.join(__dirname, '/input');
+
+describe('react-refresh', function() {
+  let b,
+    root,
+    window,
+    subscription,
+    randoms = {};
+
+  before(async () => {
+    await sleep(100);
+    await fs.rimraf(inputDir);
+    await sleep(100);
+    await fs.ncp(path.join(__dirname, '/integration/react-refresh'), inputDir);
+    let port = await getPort();
+    b = bundler(path.join(inputDir, 'index.js'), {
+      outputFS: fs,
```

Comment:
Maybe use the memory fs for input instead. Should be more reliable on Windows if possible.